Dockerfile

```diff
@@ -22,8 +22,11 @@ FROM amazoncorretto:21
 
 WORKDIR /app
 
-# Install nginx
-RUN amazon-linux-extras install -y nginx1
+# Install nginx and MySQL client
```

Comment:
Install MySQL Client in the container to support the SQL Dump feature

Reply:
It is! Since I develop locally and forgot to test the feature with postgres in containers😅, just added it, thanks!!!